content/docs/instrumenting/clientlibs.md

```diff
@@ -27,6 +27,7 @@ Unofficial third-party client libraries:
 * [Lua](https://github.com/knyar/nginx-lua-prometheus) for Nginx
 * [.NET / C#](https://github.com/andrasm/prometheus-net)
 * [Node.js](https://github.com/siimon/prom-client)
+* [PHP] (https://github.com/Jimdo/prometheus_client_php)
```

Comment:
The GitHub markdown renderer is fine with the blank in between here, but could you remove it for consistency?